dynawo/sources/API/CRV/test/TestExporters.cpp

```diff
@@ -95,22 +95,26 @@ TEST(APICRVTest, CurvesCollectionXmlExporter) {
   ASSERT_NO_THROW(curve1->update(2));
 
   // test updateParameterCurveValue (set the value to a given value)
-  ASSERT_NO_THROW(curve1->updateParameterCurveValue("variable1", 5));
+  ASSERT_NO_THROW(curve1->updateParameterCurveValue("--unused--", 5));
 
   boost::shared_ptr<Curve> curve2 = CurveFactory::newCurve();
 
   curve2->setVariable("variable2");
   curve2->setAvailable(true);
+  // Parameter curves ignore the negated flag
+  // This demonstrates that the value used in the updateParameterCurveValue method
+  // is the same value obtained when exporting, without a change in sign
   curve2->setNegated(true);
   curve2->setAsParameterCurve(false);
+  curve2->setExportType(curves::Curve::EXPORT_AS_BOTH);
```

Comment:
There should be a similar test with EXPORT_AS_CURVES

Reply:
completed the test using the `EXPORT_AS_CURVE` attribute